src/rust/vendor-rust-std.sh

```diff
@@ -0,0 +1,12 @@
+RUST_DIR="$(rustc --print=sysroot)"
+
+# The following two commands must be run as root in the docker container.
+# Copy the Cargo.lock for Rust to place the `cargo vendor` will see
+# cp "$RUST_DIR/lib/rustlib/src/rust/Cargo.lock" "$RUST_DIR/lib/rustlib/src/rust/library/test/"
+# chmod 777 $RUST_DIR/lib/rustlib/src/rust/library/test/Cargo.lock
+
+
+RUSTC_BOOTSTRAP=1 cargo vendor \
+    --manifest-path Cargo.toml \
+    --sync "$RUST_DIR/lib/rustlib/src/rust/library/test/Cargo.toml" \
```

Comment:
Could you add a comment why it's this Cargo.toml? `/test/` sounds wrong, so it could help to add some sort of reference.